### ISSUE TITLE
Clean human reads before initialisation

### ISSUE THREAD

**Issue by pbiology** [genomic-medicine-sweden/gms-artic#39]
**What needs to be done:**
Before running any computational steps on remote nodes, a cleaning of possible human contaminants should be done.

**Suggestions on how to get it done:**
We could and a pre-stage which uses a local executor (calling a local comp node), which runs cleanup of all fastq files. The question is on which software to do the cleanup. I don't have any benchmarks really, but the two that springs to mind is BBduk form the BBMap package, or kraken2. Would be great to get some input form others here. Any suggestions @talnor @sofstam @JD2112 @bokelund ?

The documentation should probably also refer to some publication showing the effectiveness of such a cleanup

**What are the arguments for getting it done:**
This way we can ensure we've taken precautions to not ending any sensitive genetic information to remote compute nodes.

**Task is considered finished when:**
All fastq files sent to remote nodes are free from human data.

**Comment by sofstam** [comment]
@pbiology What about bwa-mem?

**Comment by pbiology** [comment]
> @pbiology What about bwa-mem?

Yeah could be. I was just quick searching and had a look at this paper: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1819-8
Especially figure 1.

But I have by no means done any exhaustive investigation into this

**Comment by pbiology** [comment]
This also seems interesting:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7478626/

**Comment by bokelund** [comment]
Jag fick tips från Wolmar om denna: https://gitlab.com/uit-sfb/fhi-desensitize som norska fohm använder/utvecklat
